README.md

```diff
@@ -46,11 +46,13 @@ The supported Native platforms are:
 | iosX64            | watchosX64            | androidNativeX64   |
 |                   | watchosArm32          | watchosDeviceArm64 |
 |                   | watchosArm64          | mingwX64           |
-|                   | tvosSimulatorArm64    |                    |
+|                   | tvosSimulatorArm64    | linuxArm32Hfp      |
 |                   | tvosX64               |                    |
 |                   | tvosArm64             |                    |
 |                   | iosArm64              |                    |
 
+> :warning: The `linuxArm32Hfp` platform is deprecated
```

Comment:
Why do we want to support it, if it is deprecated?

Reply:
@ftomassetti it is deprecated because it is still under evaluation. It may be supported officially in future versions.

Also, current users of `linuxArm32Hfp` might benefit from a antlr-kotlin release.  
*If* we will remove it in the future, `linuxArm32Hfp` consumers will be able to use an older, but working, version of ANTLR.

Reply:
Ok, makes sense. I will merge this